subsys/bluetooth/controller/ll_sw/nordic/lll/lll_test.c

```diff
@@ -168,10 +169,26 @@ static int create_iq_report(bool crc_ok)
 }
 #endif /* CONFIG_BT_CTLR_DTM_HCI_DF_IQ_REPORT */
 
-static void isr_tx(void *param)
+static uint32_t calculate_tifs(void)
 {
-	uint32_t l, i, s, t;
+	uint32_t l;
+	uint32_t interval;
+
+	/* LE Test Packet Interval */
+	l = radio_tmr_end_get() - radio_tmr_ready_get();
+
+#if defined(CONFIG_BT_CTLR_DF_CTE_TX)
+	interval = ceiling_fraction((l + CTE_LEN_US(test_cte_len) + 249), SCAN_INT_UNIT_US) *
+		   SCAN_INT_UNIT_US;
+#else
+	interval = ceiling_fraction((l + 249), SCAN_INT_UNIT_US) * SCAN_INT_UNIT_US;
+#endif /* CONFIG_BT_CTLR_DF_CTE_TX */
 
+	return interval - l;
```

Comment:
Here `l` is (ready delay + PDU len), calculation is missing substracting the `radio_tx_ready_delay_get(test_phy, test_phy_flags);`